### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by francksefu**
Add Redux Toolkit (npm install react-redux @reduxjs/toolkit). -> done
Create a directory that will contain all your Redux logic (/src/redux) -> done
Configure a Redux store (/src/redux/store.js) -> done
Define a slice of state for books that:
Includes an array of books (initial state: empty array) -> done
Includes a reducer that adds a book -> done
Includes a reducer that removes a book -> done
Define a slice of state for categories that:
Includes an array of categories (initial state: empty array) -> done
Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string) -> done

### CODE REVIEW

File: src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+  },
```
Comment (IngDavidTM) [inline src/redux/store.js]: - [ ] Great job on creating the bookstore! :+1: However, as a project requirement, it is necessary to import the reducer for categories.

File: src/redux/categories/categoriesSlice.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  catList: [], status: 'Under construction',
+};
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {},
```
Comment (IngDavidTM) [inline src/redux/categories/categoriesSlice.js]: - [ ] You have done an excellent job creating the Categories Slice :+1:. However, it is a requirement of the project to include a reducer that checks the status and returns "Under construction". By including this reducer, you can help to ensure that the Categories Slice functions as expected and that the project is completed successfully. 

hint: The initial state should be and empty array and the reducer should look like this:

```js
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @francksefu  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @francksefu,

✔️ Implementation of reducers and store

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**